### PR TITLE
Add MathML to SVG Endpoint

### PR DESCRIPTION
**Pull request by zoidyzoidzoid**

### CODE REVIEW

File: mathml_to_image_service/svg_to_image.py [hunk 1]
```diff
@@ -2,45 +2,50 @@
 import subprocess
 import uuid
 
+from mathml_to_image_service.config import STATIC_DIR
```
Comment (rudigiesler) [inline mathml_to_image_service/svg_to_image.py]: Is this not supposed to be `from config import STATIC_DIR`?

File: mathml_to_image_service/service.py [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
 app = Flask(__name__)
 
 
-@app.route('/')
-def hello():
-    return 'Hello World!'
+@app.route('/', methods=['GET', 'POST'])
+def convert():
+    if request.method == 'GET':
+        return jsonify(error='Try sending MathML in a POST request. :)')
```
Comment (rudigiesler) [inline mathml_to_image_service/service.py:13]: I think we need more info, or a link to the docs, but that will be in another PR.

https://github.com/rudigiesler/mathml-to-image-service/issues/8

File: mathml_to_image_service/tests/test_service.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        self.assertEqual(
+            json.loads(self.app.get('/').data.decode('utf-8')),
+            {'error': 'Try sending MathML in a POST request. :)'})
+
+    def test_root_path_post_400_with_missing_field(self):
+        response = self.app.post('/')
+        self.assertEqual(
+            json.loads(response.data.decode('utf-8')),
+            {'error': 'Missing field: mathml'})
+        self.assertEqual(response.status_code, 400)
```
Comment (rudigiesler) [inline mathml_to_image_service/tests/test_service.py:21]: I think we need more tests, like for successful requests, and incorrect mathml, etc. But I'll do that in another PR.

https://github.com/rudigiesler/mathml-to-image-service/issues/7

File: mathml_to_image_service/service.py (hunk 2 above)
Comment (zoidyzoidzoid) [inline mathml_to_image_service/service.py:13]: Yeah. @jonathanendersby also suggested that there should be more info.

File: mathml_to_image_service/svg_to_image.py (hunk 1 above)
Comment (zoidyzoidzoid) [inline mathml_to_image_service/svg_to_image.py]: I prefer full path imports, but I see it breaks when manually running this file, and we aren't using them anywhere else, so I'll change it for consistency.

**Comment by rudigiesler** [comment]
:+1: when travis passes.

File: mathml_to_image_service/tests/test_service.py (hunk 3 above)
Comment (zoidyzoidzoid) [inline mathml_to_image_service/tests/test_service.py:21]: Yeah, these were just some basic ones that didn't rely on the other libraries.
